tests/transforms/test_metric_tensor.py

```diff
@@ -91,19 +91,6 @@ def circuit(a):
         result = qml.metric_tensor(circuit, hybrid=False, approx="block-diag")(*params)
         assert result.shape == (2, 2)
 
-    def test_generator_no_expval(self, monkeypatch):
-        """Test exception is raised if subcircuit contains an
-        operation with generator object that is not an observable"""

```

Comment:
this is removed because the new logic supports arbitrary generators?

Reply:
Yes :)